### PR TITLE
Analysis Project Command : Complete

### PR DESCRIPTION
**Pull request by jasonwalker80**
A command to complete AnPs.

### CODE REVIEW

File: lib/perl/Genome/Config/AnalysisProject/Command/Complete.pm [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+use warnings;
+
+use Genome;
+
+class Genome::Config::AnalysisProject::Command::Complete {
+    is => 'Command::V2',
+    has_input => [
+       analysis_projects  => {
+            is                  => 'Genome::Config::AnalysisProject',
+            doc                 => 'the analysis projects to release for processing',
```
Comment (tmooney) [inline lib/perl/Genome/Config/AnalysisProject/Command/Complete.pm]: This `doc` line appears to be a holdover from the `Release` command.

File: lib/perl/Genome/Config/AnalysisProject/Command/Complete.t [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    name => 'Test Project',
+    status => 'In Progress',
+);
+is($ap->status, 'In Progress', "AnP status is 'In Progress'");
+
+my $cmd = $class->execute(analysis_projects => [$ap]);
+ok($cmd->result, 'execute cmd');
+my $expected_status = 'Completed';
+is($ap->status, $expected_status, "AnP status set to '$expected_status'");
+is($cmd->status_message, 'Completed: '.$ap->__display_name__, 'Status message about releasing');
```
Comment (tmooney) [inline lib/perl/Genome/Config/AnalysisProject/Command/Complete.t]: The test description here should say `completing` rather than `releasing`.

**Comment by tmooney** [comment]
Given this, `Release`, and, `Hold`; it's looking like a good time to make a base class for these status updaters!  (`Deprecate` is different, since it has more dramatic effects.)  That can come in a subsequent PR.

**Comment by acoffman** [comment]
+1

**Comment by tmooney** [comment]
+1 :computer:

**Pull request merged**